src/adapter/graphql/src/queries/data.rs

```diff
@@ -37,6 +37,7 @@ impl DataQueries {
 
         let query_svc = from_catalog::<dyn domain::QueryService>(ctx).unwrap();
 
+        #[cfg_attr(target_pointer_width = "64", allow(clippy::cast_possible_truncation))]
```

Comment:
No, no, no. This is not going to work well.
We are casting numbers to size difference type, a very classic error from C++ world with `size_t` type.
Even though, we are not planning to build the system on 32-bit targets, this still has to be handled properly.

Generally, there are 2 ways to fix it correctly, depending on the situation:
 - avoid using `usize` type where we mean number, not a container size/index
 - use `usize` instead of number types all the way where the value means container size/index
 
Sometimes the choice would be restricted by the implementation libraries.

Reply:
Done